python/turbine_models/custom_models/stateless_llama.py

```diff
@@ -214,28 +227,62 @@ def forward(token0: torch.Tensor, *state0_flat):
     else:
         flags = [
             "--iree-input-type=torch",
-            "--iree-vm-bytecode-module-output-format=flatbuffer-binary",
             "--mlir-print-debuginfo",
             "--mlir-print-op-on-diagnostic=false",
             "--iree-llvmcpu-target-cpu-features=host",
             "--iree-llvmcpu-target-triple=x86_64-linux-gnu",
-            "--iree-llvmcpu-enable-microkernels",
-            "--iree-llvmcpu-stack-allocation-limit=256000",
             "--iree-stream-resource-index-bits=64",
             "--iree-vm-target-index-bits=64",
-            "--iree-vm-bytecode-module-strip-source-map=true",
-            "--iree-util-zero-fill-elided-attrs",
-            "--iree-vm-target-truncate-unsupported-floats",
             "--iree-codegen-check-ir-before-llvm-conversion=false",
-            "--iree-vm-bytecode-module-output-format=flatbuffer-binary",
             "--iree-opt-const-expr-hoisting=False",
         ]
-
+        if device == "cpu":
+            flags.append("--iree-llvmcpu-enable-microkernels")
+            device = "llvm-cpu"
+        elif device == "vulkan":
+            flags.extend(
+                [
+                    "--iree-hal-target-backends=vulkan-spirv",
+                    "--iree-vulkan-target-triple=" + target_triple,
+                    "--iree-spirv-index-bits=64",
```

Comment:
Have we confirmed that we need this flag as well? I figured the max allocation size should make this unnecessary.